microcosm_pubsub/consumer.py

```diff
@@ -166,8 +83,8 @@ def configure_sqs_consumer(graph):
 
     return SQSConsumer(
```

Comment:
Yes. But you want that anyway because you should *definitely* be using a different queue for email.